### PR TITLE
[feat] Cart UI and functionality

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

**Review by Avin008 — APPROVED** [review]
looks good to merge

File: src/Routes/AllRoutes.jsx [hunk 1]
```diff
@@ -4,18 +4,19 @@ import { WishList } from "../pages/WishList/WishList";
 import { Home, ProductsPage } from "../pages";
 import {Routes, Route} from "react-router-dom"
 import { LockedRoute } from "./LockedRoutes";
+import { Cart } from "../pages/Cart/Cart"
```
Comment (tusharkandpal) [inline src/Routes/AllRoutes.jsx]: Can import this from `pages` as well.

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -33,7 +35,7 @@ const Navbar = () => {
             </Link>        
             <Link to={"/cart"} className="icon-container">
 				<i className="fas fa-2xl fa-shopping-cart"></i>
-                <span className="icon-badge">2</span>
+                {isAuthenticated && cartState.length > 0 ? <span className="icon-badge">{cartState.length}</span> : null}
```
Comment (tusharkandpal) [inline src/components/Navbar/Navbar.jsx]: Instead of ternary, can use `&&`

File: src/pages/Cart/Cart.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
+main{
```
Comment (tusharkandpal) [inline src/pages/Cart/Cart.css]: Try using className

**Review by tusharkandpal — APPROVED** [review]
Code is well structured & maintained.

Suggestion: Can use barrel exports.

**Review by ganesh-kanchi — COMMENTED** [review]
[refactor] implement suggested changes

**Pull request merged**